src/domains/dashboard/components/menu-sidebar.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react"
+import classNames from "classnames"
+
+import { name2id } from "utils/name-2-id"
+import { sortObjectByPriority } from "domains/dashboard/utils/sorting"
+import { Menus } from "domains/dashboard/utils/netdata-dashboard"
+
+interface Props {
+  currentChart: string
+  menuNames: string[]
+  menus: Menus
+}
+export const MenuSidebar = ({
+  currentChart,
+  menuNames,
+  menus,
+}: Props) => {
+  const handleScrollToId = (event: React.MouseEvent) => {
+    event.preventDefault()
+    const chartNodeId = event.currentTarget.getAttribute("href")
+    if (chartNodeId) {
+      const node = document.getElementById(chartNodeId.replace("#", ""))
+      if (node) {
+        node.scrollIntoView()
+      }
+    }
+  }
+
+
+  return (
+    <div className="node-view__sidebar" role="complementary">
```

Comment:
Nice! Just for curiosity what is this `role` used here?

Reply:
@VLegakis good tip about the redux-store! I changed it to "Dashboard Charts".
The `role`s are copied from the old Dashboard, but i had to change some of them (eslint was complaining). I'm still not an expert on them, but i learned something.  Those attributes are used for screen readers and magnifiers

Reply:
Gooood. Didn't know that! thanks!